### PR TITLE
Merge pull request #46393 from ryzngard/issue/46192_nre_save_rename

### PR DESCRIPTION
**Pull request by ryzngard**
Cherry-pick fix for NRE on rename for servicing 

Original PR #46393

### CODE REVIEW

**Comment by ryzngard** [comment]
THIS SHOULD NOT BE MERGED UNTIL SERVICING DECISION IS MADE

**Review by JoeRobich — APPROVED** [review]

**Review by jasonmalinowski — APPROVED** [review]

File: azure-pipelines-integration.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
 - job: VS_Integration
   pool:
     name: NetCorePublic-Pool
-    queue: buildpool.windows.10.amd64.vs2019.pre.open
+    queue: buildpool.windows.10.amd64.vs2019.open
```
Comment (Cosifne) [inline azure-pipelines-integration.yml:21]: So the reason for changing this is because when this change is made vs2019.pre is 16.7 and now vs2019 is 16.7?

**Review by Cosifne — APPROVED** [review]

File: azure-pipelines-integration.yml (hunk 1 above)
Comment (ryzngard) [reply azure-pipelines-integration.yml:21]: Correct. We do not want to merge any of this change back into master. 16.8 already has the right things for this. We do want to test this on 16.7, which is now the released version, and integration tests won't pass on 16.8

**Comment by JoeRobich** [comment]
Passing Validation Build: https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/278954 (internal users only)

**Pull request merged**